src/components/Layout/GridTableLayout/GridTableLayout.test.tsx

```diff
@@ -37,6 +37,8 @@ const rows: Row[] = [
 ];
 
 describe("GridTableLayout", () => {
+  beforeEach(() => sessionStorage.clear());
```

Comment:
Not against this change, but wonder if we already have a test-wide level beforeEach that does this; or could move this to `setupTests` if we don't.